frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,33 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export const Menu: ComponentMultiStyleConfig = {
+  parts: ['list', 'item'],
+  baseStyle: {
+    item: {
+      padding: '0.75rem 1rem',
+      fontWeight: '400',
+      _hover: {
+        padding: '0.75rem 1rem',
+        background: 'primary.100',
+        borderWidth: '0rem',
```

Comment:
so the chakra implementation of `_hover` seems slightly buggy, and the `_focus` border is displayed on hover. So `borderWidth: '0rem'` is to override and remove the focus border on hover only

But i've removed the `borderWidth: '0rem'` override for active since (as per discussion) it should retain the focus border if applicable